fastlane/lib/fastlane/plugins/template/.rubocop.yml

```diff
@@ -0,0 +1 @@
+../../../../../.rubocop.yml
```

Comment:
That doesn't seem right? Does that require the fastlane main repo to be in that directory?